runtime/doc/options.txt

```diff
@@ -3128,7 +3128,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 	use for that occasion.  The occasions are:
 	|hl-SpecialKey|	 8  Meta and special keys listed with ":map"
 	|hl-Whitespace|	 0
-	|hl-EndOfBuffer|   ~ lines after the last line in the buffer
+	|hl-EndOfBuffer|   ~  lines after the last line in the buffer
+	|hl-NormalNC|	 I  inactive (not the current) window
```

Comment:
how nice of vim to repeat exactly the same information twice. But maybe we should delete the `'highlight'` option anyway, it seems relevant only for vim compiled without `+syntax` ...